static/styles/portico/portico-signin.scss

```diff
@@ -633,16 +633,19 @@ button.login-social-button:active {
 }
 
 .login-page-container .right-side .actions,
-.forgot-password-container .actions {
+.forgot-password-container .actions,
+.reset-password-emailed-container .actions {
     margin: 20px 0px 0px;
     text-align: left;
 }
 
-.forgot-password-container .actions {
+.forgot-password-container .actions,
+.reset-password-emailed-container .actions {
     line-height: 0;
 }
 
-.forgot-password-container .actions .back-to-login i {
+.forgot-password-container .actions .back-to-login i,
+.reset-password-emailed-container .actions .back-to-login i {
```

Comment:
Probably a better change would be to make this `dev.actions .back-to-login`, so that we can use this widget on other pages in the future without needing to change the CSS again.

(And maybe we should rename `actions` to `back-to-login-wrapper` for clarity)